src/index.html

```diff
@@ -12,6 +12,161 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="field">
+          Surname:
+          <input
+            id="surname"
+            name="surname"
+            type="text"
+            autocomplete="off">
+        </label>
+        <label for="name" class="field">
+          Name:
+          <input
+            id="name"
+            name="name"
+            type="text"
+            autocomplete="off">
+        </label>
+        <label for="years" class="field">
+          How old are You?
+          <input
+            id="years"
+            name="years"
+            type="number"
+            value="12"
+            min="12"
+            max="100">
+        </label>
+        <label for="date" class="field">
+          Full date of birth:
+          <input
+            id="date"
+            name="date"
+            type="date">
+        </label>
+        <label for="terms" class="field">
+          I accept the terms of the agreement
+          <input
+            id="terms"
+            name="terms"
+            type="checkbox"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+
+        <label for="email" class="field">
+          E-mail:
+          <input
+            id="email"
+            name="email"
+            type="email"
+            required placeholder="email@example.com">
+        </label>
+        <label for="pass" class="field">
+          Password:
+          <input
+            id="pass"
+            name="pass"
+            type="password"
+            required
+            minlength="5"
+            maxlength="30">
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you</legend>
+
+        <label for="lovecats" class="field">
+          Do you love cats?
+          <input
+            id="lovecats"
+            name="lovecats"
+            type="radio"
+            value="yes">Yes
+          <input
+            name="lovecats"
+            type="radio"
+            value="no">No
```

Comment:
```suggestion
          <input
            name="lovecats"
            type="radio"
            value="no"
          >
             No
```
Текст yes и no должен быть лейблом, чтобы при клике на него можно было выбрать нужный ответ